neurom/analysis/morphmath.py

```diff
@@ -47,6 +47,49 @@ def vector(p1, p2):
     return np.subtract(p1[0:3], p2[0:3])
 
 
+def two_points_fraction(p1, p2, fraction):
```

Comment:
Hmm, I think interpolate too general. Interpolation can be linear, polynomial,etc.

linear_interpolate maybe?